src/Hyperplane.jl

```diff
@@ -192,7 +192,7 @@ Otherwise this function throws an error.
     end
 
     first_nonzero_entry_a = -1
-    if all(d .== 0)
+    if iszero(d)
```

Comment:
👍  i didn't know of its existence at the time of https://github.com/JuliaReach/Reachability.jl/pull/373

Reply:
Oh, I already forgot about that again :astonished: